File: DSA/AadrikaBehl_590012053/Day2Question1.py
def sum_of_even_numbers():
    n = int(input("Enter the number of elements: "))
    nums = list(map(int, input("Enter the numbers separated by space: ").split()))
    
    if len(nums) != n:
        print("Error: Number of elements does not match the input n.")
        return
    
    even_sum = sum(num for num in nums if num % 2 == 0)
    print("Sum of even numbers:", even_sum)


sum_of_even_numbers()

File: DSA/Gautam_Shyamnani_590013078/Day2Question1.cpp
#include<iostream>
#include<vector>
using namespace std;
int main(){
    vector<int>nums={3,4,6,9};
    int sum=0;
    int n=nums.size();
    for(int i=0;i<n;i++){
        if(nums[i]%2==0){
            sum+=nums[i];
        }
    }
    cout<<"The sum of even numbers in the array is: "<<sum<<endl;
    return 0;
}
